<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGalleryDialog</class>
 <widget class="QDialog" name="AddGalleryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Gallery</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="plSource">
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pwSource" native="true">
     <layout class="QHBoxLayout" name="plSourceContent">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="pleSource"/>
      </item>
      <item>
       <widget class="QToolButton" name="ptbBrowse">
        <property name="icon">
         <iconset resource="../../GalleryExplorer.qrc">
          <normaloff>:/res/resources/folder.png</normaloff>:/res/resources/folder.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pwFree" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pwButtons" native="true">
     <layout class="QHBoxLayout" name="plButtons">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="ppbAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="pwBtnFree" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="ppbCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../GalleryExplorer.qrc"/>
 </resources>
 <connections/>
</ui>
